### ISSUE TITLE
A button to edit existing mob spawns in the Actions Editor

### ISSUE THREAD

**Issue by nuttapillar** [PikaMug/Quests#2095]
### What does your feature do?

A new button that lets you modify an existing mob spawn, as opposed to the existing option of just starting again. It'd be especially useful for more complex/custom mob setups with many attributes.

### How would people use it?

It would save a lot of time, as it would avoid the user having to start again if they want to make changes to an existing mob spawn.

**Comment by PikaMug** [comment]
I see how this could be useful, but it's not just mob spawns—many prompts work like this. Will have to give this some thought.

**Comment by PikaMug** [comment]
The more I think on this the more I'm convinced there isn't an easy way to do it. Even if a generic prompt for clearing list items were added, it would pretty much need to be numerical (i.e. enter # to delete) but that list could get super long depending on the number of items, something that's already a problem when adding items. A string prompt (i.e. enter "Dirt x 1") would solve that, but there could be identically-named items, and they could be tedious for the user to type (i.e. enter "Super Omega Dirt Deluxe 9000 Mark III Alpha x 64"). A clickable block of text could be placed next to each item, which would actually just run a command, but it may have to pass a considerable amount of data and I don't love the idea of adding one for this particular task.

Tl;dr: Can't think of a clean way to do this.

What might be better is to have some sort of web editor made which includes this functionality, perhaps similar to the LuckPerms web editor. There have been a few projects in the past which interface directly with the format of _quests.yml_ but the quality was generally dubious. Rather than leave this open, I'll let you decide if that's something worth looking into.
